src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -4,171 +4,120 @@
-   :General [{:name  "general"
-              :emoji "🐷"}
-             {:name     "people-ops"
-              :emoji    "🌏"
-              :locked?  true
-              :on-press #((open-token-gating-mocked
-                           "onboarding"
-                           "🌏"
-                           (colors/custom-color :blue 50)))}
-             {:name  "announcements"
-              :emoji "🎺"}]
-   :Mobile  [{:name  "mobile"
-              :emoji "👽"}
-             {:name  "mobile-ui"
-              :emoji "👽"}
-             {:name  "mobile-ui-reviews"
-              :emoji "👽"}]
-   :Desktop [{:name  "desktop"
-              :emoji "👽"}
-             {:name  "desktop-ui"
-              :emoji "👽"}
-             {:name  "desktop-ui-reviews"
-              :emoji "👽"}
-             {:name  "desktop2"
-              :emoji "👽"}
-             {:name  "desktop-ui2"
-              :emoji "👽"}
-             {:name  "desktop-ui2-reviews"
-              :emoji "👽"}]})
-
-(defn channel-list-component-fn
-  [channel-heights first-channel-height]
-  [rn/view
-   {:on-layout #(swap! first-channel-height
-                  (fn []
-                    (+ (if platform/ios?
-                         0
-                         38)
-                       (int (Math/ceil (oops/oget % "nativeEvent.layout.y"))))))
-    :style     {:margin-top 20 :flex 1}}
-   (map-indexed (fn [index category]
-                  (let [first-category (first category)]
-                    ^{:key first-category}
-                    [rn/view
-                     {:flex      1
-                      :key       (str index first-category)
-                      :on-layout #(swap! channel-heights
-                                    (fn []
-                                      (sort-by :height
-                                               (conj @channel-heights
-                                                     {:height (int (oops/oget % "nativeEvent.layout.y"))
-                                                      :label  first-category}))))}
-
-                     [quo/divider-label
-                      {:label            first-category
-                       :chevron-position :left}]
-                     [rn/view
-                      {:margin-left   8
-                       :margin-top    10
-                       :margin-bottom 8}
-                      (map-indexed (fn [inner-index channel-data]
-                                     [rn/view
-                                      {:key        (str inner-index (:name channel-data))
-                                       :margin-top 4}
-                                      [quo/channel-list-item channel-data]])
-                                   (first-category mock-list-of-channels))]]))
-                mock-list-of-channels)])
-
-(def channel-list-component (memoize channel-list-component-fn))
+  [rn/view {:height 350 :margin-top 20} ;TODO: fix size
```

Comment:
Same comment I made about the TODO. If this TODO is important, better have a separate GH issue, and consider deleting it.